www/secretary/workbench/views/actions/incomplete.json.rb

```diff
@@ -23,6 +23,7 @@
   '@unreadable_scan' => 'the scan is not readable or not complete',
   '@wrong_identity' => 'the public key does not match the name/email on the form',
   '@validation_failed' => 'gpg validation failed',
+  '@signature_not_armored' => 'gpg signature_not_armored',
```

Comment:
The text on the RHS is sent to the user, and the sample above is not exactly clear... also it's not just that the sig must be ASCII-armoured: we also need a detached signature, i.e. not a file with the signature appended.